machines/bleShare/request/requestMachine.ts

```diff
@@ -407,15 +407,15 @@ export const requestMachine =
                 },
               },
             },
-            navigatingToHome: {
+            navigatingToHistory: {
               invoke: {
                 src: 'disconnect',
               },
             },
             displayingIncomingVC: {
```

Comment:
As discussed, displayingIncomingVC will not be required, since anil is having some changes related to that in next sprint, keeping it, please remove if it not needed.